docs/sdk-and-tools/notifier.md

```diff
@@ -0,0 +1,248 @@
+- `UseAuthorization`: signals whether to use authorization. For testing purposes it can be set to `false`.
+- `ProxyUrl`: host and port on which the `eventNotifier` will push the parsed event data.
+- `Username`: the username used for authorization, if enabled.
+- `Password`: the password used for authorization, if enabled.
+
+The corresponding config section for enabling the driver:
+
+```toml
+[EventNotifierConnector]
+    # Enabled will turn on or off the event notifier connector
+    Enabled = true
+
+    # UseAuthorization signals the proxy to use authorization
+    # Never run a production setup without authorization
+    UseAuthorization = false
+
+    # ProxyUrl is used to communicate with the subscriptions hub
+    # The indexer instance will broadcast data using ProxyUrl
+    ProxyUrl = "http://localhost:5000"
+
+    # Username and Password need to be specified if UseAuthorization is set to true
+    Username = ""
+
+    # Password is used to authorize an observer to push event data
+    Password = ""
+```
+
+:::tip
+Due to the possible high data volume, it's not recommended to use validators as nodes
+to push events to Notifier Service.
+Similar to Elasticsearch indexing, our implementation uses a concept of a queue and makes
+sure that everything is being processed. Consensus and synchronization mechanisms can have
+delays due to outport driver.
+:::
+
+### Notifier Service
+
+In the notifier configuration directory (`cmd/notifier/config`), there is the `config.toml`
+file that can be used to configure the service.
+
+The supported config variables are:
+- `Port`: the port on which the http server listens on. Should be the same 
+  as the port in the `ProxyUrl` described above.
+- `Username`: the username used to authorize an observer. Can be left empty for `UseAuthorization = false`.
+- `Password`: the password used to authorize an observer. Can be left empty for `UseAuthorization = false`.
+- `CheckDuplicates`: if true, it will check (based on a locker service using redis) if the event have been already pushed to clients
+
+The `ConnectorApi` section has to be aligned with the one from observer node:
+```toml
+[ConnectorApi]
+    # The port on which the Hub listens for subscriptions
+    Port = "5000"
+
+    # Username is the username needed to authorize an observer to push data
+    Username = ""
+    
+    # Password is the password needed to authorize an observer to push event data
+    Password = ""
+
+    # CheckDuplicates signals if the events received from observers have been already pushed to clients
+    # Requires a redis instance/cluster and should be used when multiple observers push from the same shard
+    CheckDuplicates = true
+```
+
+If `CheckDuplicates` will be set to true, notifier service will try to connect to a redis 
+instance. In this context, redis will be used as a locker service mechanism for deduplication.
+This is usefull in scenarios when multiple observer nodes from same shard are being to send
```

Comment:
usefull -> useful

Reply:
to send -> sent

Reply:
changed to "are used to send", missed to include "used"